src/code/z_vr_box.c

```diff
@@ -232,7 +234,7 @@ void Skybox_Setup(GameState* gameState, SkyboxContext* skyboxCtx, s16 skyboxId)
     }
 }
 
-void func_80143324(PlayState* play, SkyboxContext* skyboxCtx, s16 skyboxId) {
+void Skybox_ManageResources(PlayState* play, SkyboxContext* skyboxCtx, s16 skyboxId) {
```

Comment:
Manage resources doesn't make sense here.
This function is similar to `Skybox_Setup` so it probably be named similarly.